zynqmp/optee-os/optee-os.bb

```diff
@@ -0,0 +1,88 @@
+OPTEE_VERSION = "3.4.0"
+SRCREV ??= "406c609bbf08407c6493082bc1cf246465c484c6"
+BRANCH ??= "master"
+
+DESCRIPTION = "OP-TEE OS"
+
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=69663ab153298557a59c67a60a743e5b"
+
+PROVIDES = "virtual/optee-os"
+
+inherit deploy
+
+DEPENDS += "u-boot-mkimage-native"
+
+S = "${WORKDIR}/git"
+B = "${WORKDIR}/build"
+
+PV = "${OPTEE_VERSION}+git${SRCPV}"
+
+BRANCH ??= ""
+REPO ??= "git://github.com/OP-TEE/optee_os.git;protocol=https"
+BRANCHARG = "${@['nobranch=1', 'branch=${BRANCH}'][d.getVar('BRANCH', True) != '']}"
+SRC_URI = "${REPO};${BRANCHARG} \
+	"
+
+OPTEE_BASE_NAME ?= "${PN}-${PKGE}-${PKGV}-${PKGR}-${DATETIME}"
+OPTEE_BASE_NAME[vardepsexclude] = "DATETIME"
+
+COMPATIBLE_MACHINE = "zynqmp"
+PLATFORM_zynqmp = "zynqmp"
+
+# requires CROSS_COMPILE set by hand as there is no configure script
+export CROSS_COMPILE="${TARGET_PREFIX}"
+
+# Let the Makefile handle setting up the CFLAGS and LDFLAGS as it is a standalone application
+CFLAGS[unexport] = "1"
+LDFLAGS[unexport] = "1"
+AS[unexport] = "1"
+LD[unexport] = "1"
+
+DEBUG ?= "1" 
+
+EXTRA_OEMAKE_append = " ${@bb.utils.contains('DEBUG', '1', ' DEBUG=${DEBUG}', '', d)}"
+
+OUTPUT_DIR = "${B}/${PLATFORM}/out/arm"
+TA_DEV_KIT_DIR = "${TMPDIR}/deploy/images/${MACHINE_NAME}/optee/export-user_ta"
+
+EXTRA_OEMAKE_append = " comp-cflagscore=--sysroot=${STAGING_DIR_HOST}"
+EXTRA_OEMAKE_append = " O=${OUTPUT_DIR}"
+EXTRA_OEMAKE_append = " CROSS_COMPILE=${CROSS_COMPILE}"
+EXTRA_OEMAKE_append = " CROSS_COMPILE_core=${CROSS_COMPILE}"
+EXTRA_OEMAKE_append = " CROSS_COMPILE_ta_arm64=${CROSS_COMPILE}"
+EXTRA_OEMAKE_append = " PLATFORM=zynqmp"
+EXTRA_OEMAKE_append = " CFG_ARM64_core=y CFG_ARM32_core=n"
+EXTRA_OEMAKE_append = " CFG_USER_TA_TARGETS=ta_arm64"
+EXTRA_OEMAKE_append = " CFG_TEE_BENCHMARK=n"
+EXTRA_OEMAKE_append = " CFG_TEE_CORE_LOG_LEVEL=3 CFG_PM_DEBUG=0"
```

Comment:
Suggestion: maybe you would like `CFG_TEE_CORE_DEBUG` be set here.

Reply:
`CFG_TEE_CORE_DEBUG=n` added but `CFG_PM_DEBUG` removed because it is deprecated.